Packs/HarfangLabEDR/IncidentFields/incidentfield-OS_Type.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Please use the existing field "OS"

Reply:
Hi Ido,

We actually have several fields to map:
- osproducttype (ex: 'Windows 10 Pro') => mapped on XSOAR "OS" field
- osversion (ex: '10.0.22000') => mapped on XSOAR "OS Version" field
- ostype (ex: 'windows') => mapped on HarfangLab EDR OS Type

We did not find a dedicated OS Type field in XSOAR standard fields. That is why we use our own field. 

Thanks,
Pierre

Reply:
Hi Pierre, you have a good point! Looks like `OS Type` is a field that other customers may want to use too actually.
If so - could you please add it as a field to the CommonTypes pack rather than the HarfangLabEDR pack? Thank you